user_guide.adoc

```diff
@@ -3402,13 +3403,13 @@ processor class should implement. For Java, it's named
   this particular example, it just adds whatever we supplied as "key"
   to every byte
 
-NOTE: `decode` can return different number of bytes that it gets
-(which is perfectly normal, for example, for decompression).
+NOTE: `decode` can return a different number of bytes that it gets
+(which is perfectly normal, for example, with decompression).
```

Comment:
I don't get this one (maybe it should be like this, but I'm not sure). Is there a rationale behind it, or your feeling?

Reply:
I've modified these lines again. Better now?